### DISCUSSION TITLE
Batch run

### DISCUSSION THREAD

**Discussion by alemiser** [hoppscotch/hoppscotch#1538] [Ideas]
How about allow user to define a sequence of API to call and record result.

**Comment by liyasthomas** [comment]
This is a good feature request. But implementation wise, instead of letting users create every single requests one by one in series within a browser tab, I guess it's better let users run every requests inside a collection / folder. So that users can easily add / remove requests in a orderly manner. What do you think?

**Reply by eg9y** [reply]
was looking exactly for this feature. Is it possible to do something like this now? Postman has this: 
![image](https://github.com/hoppscotch/hoppscotch/assets/23127928/e4bfdfb0-7e41-470a-a12a-6874728fced8)

**Reply by liyasthomas** [reply]
We're working on this feature, expect this to be released in next major release candidate scheduled for December end.

![image](https://github.com/hoppscotch/hoppscotch/assets/10395817/b8d78282-f65e-46f1-b519-f43ba2b74703)

**Reply by liyasthomas** [reply]
We're working on this feature, expect this to be released in next major release candidate scheduled for December end.

![image](https://github.com/hoppscotch/hoppscotch/assets/10395817/b8d78282-f65e-46f1-b519-f43ba2b74703)

**Reply by eg9y** [reply]
Awesome, looking forward

**Comment by Faisal-Alsumait** [comment]
Any updates?

**Comment by tanvirkhanja** [comment]
Hi Team,
Eagerly waiting for this feature.
Any update on this.

Thanks in advance.

**Comment by jamesgeorge007** [comment]
Hi there, the GUI edition of Collection Runner is now available on all Hoppscotch platforms, including the Desktop app. You can learn more from the documentation available [here](https://docs.hoppscotch.io/documentation/features/runner).
